yt_dlp/extractor/rokfin.py

```diff
@@ -195,7 +195,7 @@ class RokfinStackIE(RokfinPlaylistBaseIE):
     def _real_extract(self, url):
         list_id = self._match_id(url)
         return self.playlist_result(self._get_video_data(
-            self._download_json(f'{_API_BASE_URL}stack/{list_id}', list_id)), list_id)
+            self._download_json(f'{_API_BASE_URL}stack/{list_id}', list_id)), list_id, multi_video=True)
```

Comment:
The example in TESTS is definitely not a multi_video
```suggestion
            self._download_json(f'{_API_BASE_URL}stack/{list_id}', list_id)), list_id)
```